### DISCUSSION TITLE
Where can I download the largest template containing all languages?

### DISCUSSION THREAD

**Discussion by AppleFlan** [openai/whisper#1151] [Q&A]
I am using MacWhisper to transcribe audios, and I want to try using other larger models to see how much the results change. Sorry for my ignorance I like to mess around, but I'm new to Github.

**Comment by sscotti** [comment]
If you have the resources, you might want to setup your own environment using something like Docker containers with the Whisper API baked in.  It would make it much easier to develop and play around with models.  There are plenty of resources about that on the internet:

e.g.

[OpenAI Whisper tutorial: How to use OpenAI Whisper](https://lablab.ai/t/whisper-tutorial)
[ChatGPT in your Laravel apps](https://filippotoso.medium.com/chatgpt-in-you-laravel-apps-the-easy-way-c8c1aceba5cf)
[OpenAI Whisper tutorial: Creating OpenAI Whisper API in a Docker Container](https://lablab.ai/t/whisper-api-flask-docker)
